### ISSUE TITLE
[examples/sveltekit-scoped/] Trying to update the example with `preset-typography` without much luck.

### ISSUE THREAD

**Issue by bySabi** [unocss/unocss#1920]
Hello @jacob-8 I have made a fork of the example with some minimal changes that I describe in the README header.

Here is the repo: https://github.com/bySabi/example-unocss-sveltekit-scoped

I think the problem is that the `preset-typography` doesn't get loaded globally.
 
The _Visual Studio Code UnoCSS_ extension also does not highlight the `prose` class

It would be nice to have a `preset-typography` example with SvelteKit. I am not able to find any.

**Comment by jacob-8** [comment]
@bySabi thanks for your patience on this - you've probably moved on by now, but do know that #2098 includes the ability to use `presetTypography` as well as a usage example. Once that is merged, you'll be good to go.

**Comment by jacob-8** [comment]
@bySabi we've created a new project that depends on UnoCSS called [svelte-scoped-uno](https://github.com/jacob-8/svelte-scoped-uno). You can read how to install at https://github.com/jacob-8/svelte-scoped-uno/blob/main/packages/svelte-scoped-uno/README.md and you have an example project with typography at https://github.com/jacob-8/svelte-scoped-uno/blob/main/examples/sveltekit-vite-plugin - note especially the [unocss.config.ts](https://github.com/jacob-8/svelte-scoped-uno/blob/main/examples/sveltekit-vite-plugin/unocss.config.ts)
